inspector/transform.py

```diff
@@ -1,7 +1,10 @@
+import json
 import os
 import re
 import requests
 
+from .geekbench import geekbench_html_to_json
```

Comment:
```suggestion
from geekbench import geekbench_html_to_json
```
This is just a plain python script, not a module, so this must be used.